isogeny_primes.py

```diff
@@ -824,21 +879,23 @@ def get_possible_vals_at_gens(gens_info, eps, embeddings, residue_field, prime_f
         class_gp_order, alpha = gens_info[class_gp_gen]
         alpha_to_eps = eps_exp(alpha, eps, embeddings)
         alpha_to_eps_mod_p0 = residue_field(alpha_to_eps)
-
+        logging.debug(f"alpha^eps mod p0: {alpha_to_eps_mod_p0}")
         try:
             c_power_12h = prime_field(alpha_to_eps_mod_p0)
-            possible_values = c_power_12h.nth_root(12*class_gp_order, all=True)
-            filtered_values = filter_possible_values(possible_values)
-            output[class_gp_gen] = list(set([x**12 for x in filtered_values]))
-        except TypeError:
+        except TypeError as err:
```

Comment:
@BarinderBanwait  This was the reason why the filtering was going wrong. This "except TypeError" was spanning multiple commands. And there was a bug in on of those commands causing it to raise a TypeError totally unrelated to alpha_to_eps_mod_p0 being in prime_field.

Reply:
oh dear, yeah you pointed this out in a previous review as well, i fixed one of these "generous catching of errors", but forgot about this one; well spotted here! will try to fix the lower `filter_possible_values`

Reply:
`filter_possible_values` now has an explicit sanity to check to only take weil polys forward. also changes to `lifts_in_range`  should hopefully have fixed everything here.